script/update_samples.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SAMPLES_REV="1.0"
+
+TMP_DIR="${DIR}/../build"
+SAMPLES_DIR="${DIR}/../samples"
+
+cleanup() {
+    rm -rf "${DIR}/../build/biscuit_spec"
+}
+
+#trap "cleanup" ERR
```

Comment:
Should this be commented? In general we should remove lines if they are not used.

Reply:
Ah thanks, it's meant to be uncommented yep